### ISSUE TITLE
Issue Documenting Array of objects

### ISSUE THREAD

**Issue by RajviPJain** [knuckleswtf/scribe#892]
### Scribe version

4.37

### PHP version

8.1

### Framework

Laravel

### Framework version

9.19

### Scribe config

```ruby
title => "SCRIBE"
type => "laravel"
laravel.add_routes => false
auth.enabled => true
auth.placeholder => "token"
strategies.responses => added [
    'Knuckles\\Scribe\\Extracting\\Strategies\\Responses\\ResponseCalls',
    [
        'only' => [
            'GET *',
        ],
    ],
]: removed [
    'Knuckles\\Scribe\\Extracting\\Strategies\\Responses\\ResponseCalls',
    [
        'only' => [
            'GET *',
        ],
        'config' => [
            'app.debug' => false,
        ],
    ],
]
```

### What happened?

I have used the following code comments 

/**
* @bodyParam user object required The user details
* @bodyParam user.name string required The user's name
* @bodyParam user.age string required The user's age
* @bodyParam friend_ids int[] List of the user's friends.
* @bodyParam cars object[] List of cars
* @bodyParam cars[].year string The year the car was made. Example: 1997
* @bodyParam cars[].make string The make of the car. Example: Toyota
*/

But it is not generating docs as per given output in docs

Given Output
![image](https://github.com/user-attachments/assets/8acb5710-7d90-43e6-bcef-ac4552467bc6)

Actual Output
![image](https://github.com/user-attachments/assets/7c9ab959-2bdd-4e65-92dd-6491066fa79a)

### Docs

- [X] I've checked [the docs](https://scribe.knuckles.wtf/laravel), the [troubleshooting guide](https://scribe.knuckles.wtf/laravel/troubleshooting), and [existing issues](https://github.com/knuckleswtf/scribe/issues?q=is%3Aissue+), but I didn't find a solution

**Comment by Felipe-Rafael-Rosa** [comment]
try this
https://bit.ly/3XQLHlH

you may need to install the c compiler

**Comment by shalvah** [comment]
I don't think this is a bug. That's just an old screenshot. We don't show the full name anymore (`cars[].year`), since it's now visually nested under `cars`.

**Comment by RajviPJain** [comment]
Thank you for the clarification. However, I still believe the previous visual representation was **clearer**. For example, even if the user names the variable `car`, the nested variable would appear as `car[].year.` This format explicitly indicates that `car` is an array of objects, which enhances understanding for other users.

It would be helpful to revert to the old visual representation. If that's not feasible, I suggest updating the current documentation to align with the example output, as it currently may lead to confusion.

Thank you for considering my feedback!

**Comment by RajviPJain** [comment]
I don't see any option to reopen this issue

@shalvah can you please reopen this issue if possible at your end?

Thanks
